### ISSUE TITLE
Lilu DeviceInfo detection precludes MacHyperVSupport PCI bridges

### ISSUE THREAD

**Issue by flagersgit** [acidanthera/bugtracker#2299]
As title mentions, Lilu's DeviceInfo class does not consider MacHyperVSupport PCI bridges.

This is understandably so, because the Hyper-V Gen 2 VM topology is rather weird and nothing alike a normal PC, but nonetheless still problematic.

For example, GPUs (and accompanying HDA) passed to VMs will not have the benefit of patches being applied by WhateverGreen and AppleALC.

**Comment by vit9696** [comment]
Fair enough, but given MacHyperVSupport starts late I really wonder whether we actually can support these. Most likely Lilu will not see these devices. Have you tried to patch Lilu to support this?

**Comment by flagersgit** [comment]
I haven't tried to patch Lilu yet.

I was considering some approaches to solving this problem. 
Perhaps Lilu can route IOPCIBridge::publishNub to see the devices as soon as possible (immediately after they are published children of the HyperVPCIBridge). Will, of course, have to test and see.
